### DISCUSSION TITLE
Notable Events Page

### DISCUSSION THREAD

**Discussion by leahcimto** [theforumhelpers/theforumhelpers.github.io#363] [Ideas]
We are discussing the creation of a notable events page for our website. However, to do so we need some events! Notable events are things like important special actions or moving studios/hosts. Please share and discuss ideas here. Thank you!

The issue suggesting the page is here: #358 

Edited by @Accio1: For clarity, please do not suggest things such as "drama" or debates/disagreements. This page should be for monumental events, not for flame wars that tarnish the studio's reputation.

**Comment by somerandomdood3** [comment]
maybe add the results of the studio host vote? like how cats united won with #, 2nd up was __ with #, etc.

**Reply by penguinmoose** [reply]
I don't think that's a very important detail, although we could add it in case some people want to know.

**Comment by Accio1** [comment]
For clarity, please do not suggest things such as "drama" or debates/disagreements. This page should be for monumental events, not for flame wars that tarnish the studio's reputation.

**Comment by leahcimto** [comment]
This may be helpful as well: https://en.scratch-wiki.info/wiki/User:Sharkode/TFH_History
